spec/features/embargo_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.feature 'embargo' do
+  let(:user) { create(:user) }
+  before do
+    sign_in user
+  end
+  describe 'creating an embargoed object' do
+    before do
+      visit '/'
+    end
+
+    let(:future_date) { 5.days.from_now }
+    let(:later_future_date) { 10.days.from_now }
+
+    # Pending revision to work in hyrax
+    xit 'can be created, displayed and updated', :workflow do
```

Comment:
What revision is necessary? Should that be ticketed and at least left as a hint to the next person?